src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,27 @@
 import React from 'react';
+import { Comment } from '../../types/Comments';
+import { CommentInfo } from '../CommentInfo/CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
-);
+export interface Allcoments {
+  comments: Comment[];
+}
+
+export const CommentList: React.FC<Allcoments> = ({ comments }) => {
+  return (
+    <>
+      {comments.length < 1
+        ? (
+          <b data-cy="NoCommentsMessage">No comments yet</b>
+        )
+        : (
```

Comment:
```suggestion
      {comments.length ? (
          <b data-cy="NoCommentsMessage">No comments yet</b>
        ) : (
```

Reply:
I try take out separately [?] and [:]. But if need, okay.